svg.js.d.ts

```diff
@@ -548,13 +548,15 @@ declare module "@svgdotjs/svg.js" {
         to(a: any): Morphable;
     }
 
+    type ListEachCallback<T> = (el: T, index: number, list: T[]) => T
+
     // List.js
-    interface List<T> extends BuiltInArray<T> {
+    class List<T> extends BuiltInArray<T> {
         // I have no clue how to deal with this
         // [key: string]: (...arg0: any[]) => List<T>

```

Comment:
In case you have an idea how to make this part work... its basically all methods of everything on the List e.g. `list.move(10, 10).animate().move(40, 40)`

Reply:
See above.